css/main.css

```diff
@@ -60,6 +60,24 @@ li {
     color: #fd971f;    
 }
 
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    overflow: auto;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+}
+
+.dropdown-content a {
+    color: black;
```

Comment:
As good practice try to avoid "names" in css, always go for the hex code, here you should do `#000000`